<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winDetect</class>
 <widget class="QMainWindow" name="winDetect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coin Detect</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>filename</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlButtons">
      <item>
       <widget class="QLabel" name="lblImage">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlImage">
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="text">
         <string>&lt; Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblValue">
        <property name="text">
         <string>Total Value:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>Next &gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
